GPU smoke test expects pre-#498 slurm filename

## Problem

The Nightly **GPU Smoke Test** workflow has been failing since v0.3.2 (most recent: [run 26446995657](https://github.com/niznik-dev/cruijff_kit/actions/runs/26446995657)).

\`\`\`
AssertionError: capitalization_epoch0.slurm not generated
  File "tests/integration/gpu/smoke_test_scaffolding.py", line 97, in test_setup_inspect
\`\`\`

## Cause

PR #498 (per-cell eval layout) changed \`setup_inspect.py\`'s default output filename from \`{task}_epoch{N}.slurm\` to \`cell.slurm\`:

\`\`\`python
# src/tools/inspect/setup_inspect.py:379
output_path = cli_args.output_slurm or "cell.slurm"
\`\`\`

The unit tests in \`tests/unit/test_setup_inspect.py\` were updated in #513, but the GPU integration smoke test at \`tests/integration/gpu/smoke_test_scaffolding.py:96-97\` was missed.

## Fix

One-line change: update the assertion in \`test_setup_inspect\` (within the GPU smoke test file) to look for \`cell.slurm\` instead of \`capitalization_epoch0.slurm\`.

Other workflows (Nightly Inspect Smoke Test, Nightly Torchtune Smoke Test, Nightly Tests) are unaffected.

--- a/tests/integration/gpu/smoke_test_scaffolding.py
+++ b/tests/integration/gpu/smoke_test_scaffolding.py
@@ -93,8 +93,8 @@ def test_setup_inspect(work_dir: Path):
         print(f"STDERR: {result.stderr}")
         raise RuntimeError(f"setup_inspect.py exited with code {result.returncode}")
 
-    slurm = work_dir / "capitalization_epoch0.slurm"
-    assert slurm.exists(), "capitalization_epoch0.slurm not generated"
+    slurm = work_dir / "cell.slurm"
+    assert slurm.exists(), "cell.slurm not generated"
     print("PASS: setup_inspect.py scaffolding")
 
 

